src/gui/update.rs

```diff
@@ -2134,6 +2134,7 @@ async fn perform_fetch_latest_addon(
                     catalog::Source::Tukui => RepositoryKind::Tukui,
                     catalog::Source::WowI => RepositoryKind::WowI,
                     catalog::Source::TownlongYak => RepositoryKind::TownlongYak,
+                    catalog::Source::Other => panic!("Unspported catalog source"),
```

Comment:
Should be fine to `panic!` since it shouldn't be possible to end here with a `catalog::Source::Other`.